assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -270,10 +271,14 @@ def run(args):
         log_time_and_message("Reading data in spark and converting to pandas")
         baseline_df = try_read_mltable_in_spark_with_error(args.baseline_data, "baseline_data")
 
+        raw_categorical_features = get_categorical_cols_with_df_with_override(baseline_df,
+                                                   override_numerical_features=args.override_numerical_features,
+                                                   override_categorical_features=args.override_categorical_features)
         baseline_df = baseline_df.toPandas()
         task_type = determine_task_type(args.task_type, args.target_column, baseline_df)
         log_time_and_message(f"Computed task type is {task_type}")
 
+        # Todo: get this from raw_categorical_features (removing target column, validate feature types)
         categorical_features = compute_categorical_features(baseline_df, args.target_column)
```

Comment:
Is this compute_categorical_features() written by Shreeya with some complex logic? I suppose with feature_override, we'll replace this with get_categorical_cols_with_df_with_df_with_override()?

Reply:
done the changes